components/storage-sync/sql/create_schema.sql

```diff
@@ -0,0 +1,27 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-- This is a very simple schema.
+CREATE TABLE IF NOT EXISTS moz_extension_data (
```

Comment:
Currently ext-storage does a structural merge on the items in the case of conflicts. I'd recommend we store a mirror, but given this we probably need more than just this.

Reply:
Can you please expand on the "need more than just this"? IIUC, we merge only at the top-level - so we don't need to merge "inside" object other than the top-level 1. IIUC, we could do this fairly easily with a mirror, but I haven't thought too much about it.

Reply:
Well, tombstones specifically. Probably in addition to a mirror? This depends on how things work in practice though.